src/pyedb/dotnet/edb_core/siwave.py

```diff
@@ -1429,6 +1434,51 @@ def place_voltage_probe(
         n_terminal = self._pedb.get_point_terminal(name + "_ref", negative_net_name, negative_location, negative_layer)
         return self._pedb.create_voltage_probe(p_terminal, n_terminal)
 
+    def create_vrm_module(
+        self,
+        name=None,
+        is_active=True,
+        voltage="3V",
+        positive_sensor_pin=None,
+        negative_sensor_pin=None,
+        load_regulation_current="1A",
+        load_regulation_percent=0.1,
+    ):
+        """Create a voltage regulator module.
+        Parameters
```

Comment:
```suggestion

        Parameters
```
A line break is required